addons/mass_mailing/static/src/iframe/mass_mailing_iframe.xml

```diff
@@ -0,0 +1,51 @@
+<templates>
+    <t t-name="mass_mailing.MassMailingIframe">
+        <div class="o_mass_mailing_iframe_wrapper"
+            t-att-class="{'o_mass_mailing_fullscreen': state.showFullscreen}"
+            t-attf-class="#{props.extraClass}">
```

Comment:
bit weird to have an `attf` to only include a `#{}` inside it.
If it's because we can't have 2 `t-att-class`, we can maybe do 
`t-att-class="{'o_mass_mailing_fullscreen': state.showFullscreen, props.extraClass: true}` or maybe even simpler idk